### PR TITLE
updpates to conditional display

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Look forward to advice.

### CODE REVIEW

File: talentmap_api/fsbid/services/client.py [hunk 1]
```diff
@@ -97,7 +97,8 @@ def get_client_csv(query, jwt_token, rl_cd, host=None):
         fsbid_clients_to_talentmap_clients_for_csv,
         "/api/v1/fsbid/client/",
         host,
-        ad_id
+        ad_id,
+        currentAssignmentOnly
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py]: This won't do anything

File: talentmap_api/fsbid/services/client.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def client(jwt_token, query):
     Get Clients by CDO
     '''
     ad_id = jwt.decode(jwt_token, verify=False).get('unique_name')
-    uri = f"CDOClients?request_params.ad_id={ad_id}&{convert_client_query(query)}"
+    uri = f"CDOClients?request_params.ad_id={ad_id}&request_params.currentAssignmentOnly=true&{convert_client_query(query)}"
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py]: change from true to false

File: talentmap_api/fsbid/services/client.py [hunk 3]
```diff
@@ -80,7 +80,7 @@ def single_client(jwt_token, perdet_seq_num):
     Get a single client for a CDO
     '''
     ad_id = jwt.decode(jwt_token, verify=False).get('unique_name')
-    uri = f"CDOClients?request_params.ad_id={ad_id}&request_params.perdet_seq_num={perdet_seq_num}"
+    uri = f"CDOClients?request_params.ad_id={ad_id}&request_params.perdet_seq_num={perdet_seq_num}&request_params.currentAssignmentOnly=false"
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/client.py]: change from false to true

File: talentmap_api/fsbid/services/client.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py]: 💔

**Comment by mjoyce91** [comment]
@elizabeth-jimenez I realized we'll need an update on the mock to get this to work, since it's not currently returning position data along with each assignment in the full assignment history.

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez I realized we'll need an update on the mock to get this to work, since it's not currently returning position data along with each assignment in the full assignment history.

Ok, that's fine! What's the procedure here? Do I keep this here and get back to it when the update for the mock comes through?

**Comment by mjoyce91** [comment]
@elizabeth-jimenez ultimately I ended up using this same change in #313. We can close this PR

**Pull request closed without merging**